x-pack/plugins/ml/common/util/anomaly_utils.ts

```diff
@@ -168,17 +175,17 @@ export function getSeverityWithLow(normalizedScore: number): SeverityType {
 // for the supplied normalized anomaly score (a value between 0 and 100).
 export function getSeverityColor(normalizedScore: number): string {
   if (normalizedScore >= ANOMALY_THRESHOLD.CRITICAL) {
```

Comment:
The `>` in the legend should actually be `>=` to match these conditions? Could the condition in the legend labelling be changed in a follow-up?

Reply:
thanks for spotting this! yes, I'll adjust the legend labels in a follow-up PR.